packages/ui/primitives/document-flow/add-signers.tsx

```diff
@@ -225,7 +228,9 @@ export const AddSignersFormPartial = ({
         <DocumentFlowFormContainerActions
           loading={isSubmitting}
           disabled={isSubmitting}
-          onGoBackClick={documentFlow.onBackStep}
+          onGoBackClick={
```

Comment:
If we make the change in the place I suggested above in the review we can circumvent the logic here if we wish to.

Reply:
That’s fine, right now it’s not doing that though since disabled is only depending on the submission state so I guess we’d need to change that

Reply:
I've addressed the above review (**edit-document.tsx**, first comment) of checking document status to ensure we aren't able to go back when we are out of draft mode. It does the work perfectly.